containers/Containerfile.podman

```diff
@@ -0,0 +1,166 @@
+# syntax=docker/dockerfile:1.6
```

Comment:
this file is almost a duplicate of the other one. could we auto-generate them if we need them both or have some other way of unifying them?